docs/rst/user_manual/application_menu.rst

```diff
@@ -57,6 +57,20 @@ Export to CSV
 Export all the data displayed in the current Fast DDS Monitor session to a CSV file.
 Please refer to section :ref:`export_data` for more information on the format of the generated CSV file.
 
+.. _dump_button:
```

Comment:
Add functionality to `forthcoming_versions`. It may be created such version, take a look to other repos and how we handle versions there